### PR TITLE
Tools/map_gen: Use list comprehension to replace multi line for loop.

### PR DESCRIPTION
**Pull request by redfuture**
In the old code it's missing f.close when using open.This patch use
with statement when open file.

### CODE REVIEW

File: modules/tools/map_gen/map_gen.py [hunk 1]
```diff
@@ -92,14 +92,9 @@ def create_lane(map, id):
...
 fpath = sys.argv[1]
-f = open(fpath, 'r')
-points = []
-for line in f:
-    line = line.replace("\n", '')
-    data = line.split(',')
-    x = float(data[0])
-    y = float(data[1])
-    points.append((x, y))
+with open(fpath, 'r') as f:
```
Comment (lianglia-apollo) [inline modules/tools/map_gen/map_gen.py]: Complicated comprehensions or generator expressions can be hard to read. We don't recommend expressions that cannot be fit in one line. Please see ```https://github.com/google/styleguide/blob/gh-pages/pyguide.md``` for details.

Comment (redfuture) [reply modules/tools/map_gen/map_gen.py]: got that, thanks for pointing that out , I will submit another pull request to just use with statement with open file.

Comment (redfuture) [reply modules/tools/map_gen/map_gen.py]: please close this pull request.

Comment (lianglia-apollo) [reply modules/tools/map_gen/map_gen.py]: Thanks!

**Pull request closed without merging**